src/assets/i18n/pt-BR.json5

```diff
@@ -139,7 +127,7 @@
   "admin.registries.bitstream-formats.edit.description.label": "Descrição",
 
   // "admin.registries.bitstream-formats.edit.extensions.hint": "Extensions are file extensions that are used to automatically identify the format of uploaded files. You can enter several extensions for each format.",
-  "admin.registries.bitstream-formats.edit.extensions.hint": "Extensões são extensões de arquivo que são usadas para identificar automaticamente o formato dos arquivo enviados. Você pode informar várias extensões para cada formato.",
+  "admin.registries.bitstream-formats.edit.extensions.hint": "Extensões são extensões de arquivo usadas para identificar automaticamente o formato dos arquivo enviados. Você pode informar várias extensões para cada formato.",
```

Comment:
Despite the English source phrase. I think `Extensions` redundancy should be rewritten. I would prefer something more like:
`Extensões de arquivo são usadas...`

Reply:
I agree with that.  I suggest the same for English also.  Something like `File extensions are used to...`